util/file.go

```diff
@@ -253,20 +256,68 @@ func JoinTerraformModulePath(modulesFolder string, path string) string {
 	return fmt.Sprintf("%s//%s", cleanModulesFolder, cleanPath)
 }
 
-// Returns true if the given path contains the given folder name.
-//
-// Examples:
-//
-// PathContains("/foo/bar", "foo") => returns true
-// PathContains("/foo/bar", "baz") => returns false
-func PathContains(path string, folderName string) bool {
-	pathParts := strings.Split(path, string(filepath.Separator))
+type fileManifest struct {
+	Path       string
+	encoder    *gob.Encoder
+	fileHandle *os.File
+}
 
-	for _, pathPart := range pathParts {
-		if pathPart == folderName {
-			return true
+// Clean will remove all files specified in the manifest
+func (f *fileManifest) Clean() error {
```

Comment:
Please don't use single-letter variable names. They are hard to follow more than 1-2 lines away from the declaration.